ai:
    hit_buoy:
        track_buoy:
            errorGoal: 0.1
            distGoal: 0.05
            yaw_speed_factor: -20.0
            dive_speed_factor: -1.0
            center_offset: 0.25
        ram_speed: 0.4
        ram_time_seconds: 4.0
        reset_speed: -0.8
        reset_time_seconds: 10.0
    find_buoy:
        errorGoal: 0.1
        yaw_speed_factor: -25.0
# variables below contain values for gate/nav_channel tasks
    gate_task:
        forward_time_seconds: 9.5 # how many seconds to move forward
        forward_speed: 6.0 # how fast to move forward
    move_to_gate:
        forward_speed: 10.0 # how fast to move forward
    search_gate:
        yaw_speed_factor: 3.0 # how fast to yaw while searching gate
    center:
        error_goal: 0.08 # what is the error Goal, in precentage so 0.1 is 10%
        yaw_factor: -2.0 # yaw factor for centering. Negative is to the right
        dive_factor: -2.0 # dive factor for centering. Negative is to dive
    move_forward:
        error_goal: 0.07 #What is the error Goal in moving forward
        distanceGoal: 0.03 # What is the distanceGoal, teremined from testing
        forward_speed: 0.7 # What is the speed with which we need to approach
    preq_forward:
        forward_time_seconds: 3 # requires testing
        forward_speed: 8.0 # requires testing
    strafe: # values for strafe state
        time_seconds: 7
        speed: -4.0
    turn: # values for turn state
        time_seconds: 8
        speed: 4
    experiment: # values for experimental state that 'drifts' around boy
        time_seconds: 6.5
        yaw_relative_degrees: 60
        forward_speed: 5
    move_forward_single:
        distanceGoal: 0.007
    blind_gate_task:
        forward_time_seconds: 25 # number of seconds to move forward
        forward_speed: 6.0 # speed to move forward
        yaw_error_degrees: 2 # yaw error in degrees that will be tolerated
